ui/app/styles/helper-classes/other.scss

```diff
@@ -84,18 +84,16 @@
   overflow: hidden;
 }
 
-// Screen Readers only
-// ARG TODO: we have this class in the app/styes/app.scss file. See if we can consolidate.
+// screen reader only 
 .sr-only {
-  border: 0 !important;
```

Comment:
this chunk I copied from the vault.css (which compiled our class name with a bulma class name). I merged the two.